src/Symfony/Component/Mailer/Bridge/Amazon/Transport/SesApiTransport.php

```diff
@@ -61,6 +61,7 @@ protected function doSendApi(SentMessage $sentMessage, Email $email, Envelope $e
                 'Content-Type' => 'application/x-www-form-urlencoded',
             ],
             'body' => $payload = $this->getPayload($email, $envelope),
+            'timeout' => $this->timeout * 1000,
```

Comment:
why `* 1000`? the client option should be in seconds, and so is `$this->timeout`, no?

Reply:
oh .. i thought this was in ms :laughing: